### PR TITLE
Bundle built-in types slice files into the extension (#20)

### LINKED ISSUE #20: Language server ignores / doesn't use built-in types
With version 4.1.0.

I am using an `IceRpc::ServiceAddress`. The code compiles fine but the Language Server is not happy:

```
[{
	"resource": "/C:/Users/bernard/source/repos/icerpc-ice-csharp-interop/tests/Interop.Tests/Slice/TagTests.slice",
	"owner": "_generated_diagnostic_collection_name_#5",
	"code": "E049",
	"severity": 8,
	"message": "no element with identifier 'IceRpc::ServiceAddress' exists",
	"source": "slicec",
	"startLineNumber": 18,
	"startColumn": 45,
	"endLineNumber": 18,
	"endColumn": 67
}]
```

### PR DESCRIPTION
**Pull request by ReeceHumphreys**
Closes #20 

This resolves the problem of getting errors and warnings when using built in types by shipping the built in types with the extension.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
         "script": "watch"
       },
       "env": {
-        "SERVER_PATH": "${workspaceRoot}/server/target/"
+        "SERVER_PATH": "${workspaceRoot}/server/target/",
+        "SLICE_PATH": "${workspaceRoot}/slice/"
```
Comment (externl) [inline .vscode/launch.json:21]: Maybe this should be `BUILT_IN_SLICE_PATH` for clarity?

File: .vscodeignore [hunk 2]
```diff
@@ -12,3 +12,7 @@ client/**
 server/**
 !server/target/*/release/slice-language-server
 !server/target/*/release/slice-language-server.exe
+
+# Slice
+slice/**/Internal
```
Comment (externl) [inline .vscodeignore:17]: Shouldn't this be picked up by `server/**`?

File: .vscodeignore [hunk 3]
```diff
@@ -12,3 +12,7 @@ client/**
...
+slice/**/!(*.slice)
```
Comment (externl) [inline .vscodeignore:18]: I didn't know this syntax existed. Is it the same as doing `!ice/**/*.slice`?

File: client/src/extension.ts [hunk 4]
```diff
@@ -107,6 +107,9 @@ export async function activate(context: ExtensionContext) {
     const serverPath = isProduction
       ? context.extensionPath + process.env.SERVER_PATH
       : process.env.SERVER_PATH + "debug/slice-language-server";
+    const slicePath = isProduction
```
Comment (externl) [inline client/src/extension.ts:110]: ditto. `builtInSlicePath`

**Review by externl — APPROVED** [review]
Looks good. See comments.

**Review by InsertCreativityHere — APPROVED** [review]
Looks good!

Very minor thing: we might want to add an option for disabling this.
Just on the small chance a user explicitly doesn't want to pull these files in.

Maybe it triggers a redefinition, or they don't want to accidentally want to pull in a dependency on our Slice files.

File: .vscode/launch.json (hunk 1 above)
Comment (InsertCreativityHere) [reply .vscode/launch.json:21]: I agree that would be a clearer name.

File: .vscodeignore (hunk 3 above)
Comment (ReeceHumphreys) [reply .vscodeignore:18]: I would think so but I couldnt get `!slice/**/*.slice` to work

**Pull request merged**